docs/source/types.rst

```diff
@@ -16,8 +16,72 @@ Available content types in a Plone site can be listed and queried by accessing t
    :language: http
 
 
+The API consumer can create, read, update, and delete a content-types schema.
+
+======= =============================================== =====================================================================
+Verb    URL                                             Action
+======= =============================================== =====================================================================
+POST    ``/@types/Document``                            Add field/fieldset to content type schema
+GET     ``/@types/Document``                            Get the schema of a content type
+PATCH   ``/@types/Document``                            Update existing schema fields/fieldsets properties
+PUT     ``/@types/Document``                            Replace content-type schema
```

Comment:
If I am not mistaken we do not support PUT in the other endpoints yet. I never implemented that because you can do a full PATCH instead, so it is kind of a duplicate.

Reply:
Indeed, still here we have a different use-case as with PATCH you can't delete `fields/fieldsets`. With PUT you are sure that the JSON is the full representation of the object.

This is very useful when you want to completely build the schema on the front-end side and at the end save it on the back-end.